### PR TITLE
faker.seed(0) should not be ignored

### PR DESCRIPTION
**Pull request by bwiklund**
`faker.seed(0)` is currently ignored, it probably shouldn't be?

### CODE REVIEW

File: lib/random.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var mersenne = require('../vendor/mersenne');
  */
 function Random (faker, seed) {
   // Use a user provided seed if it exists
-  if (seed) {
+  if (seed != null) {
```
Comment (Marak) [inline lib/random.js]: That doesn't seem right.

Comment (bwiklund) [reply lib/random.js]: ah, one sec, let me put this in the right place

Comment (bwiklund) [reply lib/random.js]: wait no, this seems right, this is called straight from `Faker.prototype.seed`. this line is supposed to allow 0 to be used a seed, while throwing away null/undefined

Comment (bwiklund) [reply lib/random.js]: @Marak does this make sense?

**Comment by Marak** [comment]
Thank you for your contribution @bwiklund 

Closing for now since there are no additional tests.

I can see why this would be confusing for a user if `0` is ignored, but we should dig a bit deeper and make sure this will work as intended.

Please feel free to re-open with test.

**Pull request closed without merging**